docs/guide/adding_noise.md

```diff
@@ -0,0 +1,239 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.2
+kernelspec:
+  name: python3
+  display_name: Python 3 (ipykernel)
+  language: python
+---
+
+# Adding Noise
+
+## Motivation
+
+Noise is an essential component of QEC workflows.
+The whole idea of QEC only makes sense if qubits are not perfect.
+Decoders try to make logical qubits robust against errors,
+but in order for them to do a good job we have to inform them which errors they should expect.
+
+While exploring QEC codes or designing a new decoder,
+you should be able to validate it on real data.
+In the best case, you may run an experiment on a real QPU.
+But even if you have access to a QPU, it typically spends
+more time calibrating than doing computation.
+That is why you are interested in realistic QEC simulation.
```

Comment:
```suggestion
Whether you are exploring error-correcting codes or designing a new decoder,
it's important to have realistic data to validate them.
You can obtain those data from a real QPU, however, this might be expensive or time consuming, due to requiring calibration. This is where realistic QEC simulation comes in.
```